packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -184,6 +186,20 @@ function ListViewBlockSelectButton(
 							</span>
 						</Tooltip>
 					) }
+					{ images.length ? (
+						<span className="block-editor-list-view-block-select-button__images">
+							{ images.map( ( image, index ) => (
+								<span
+									className="block-editor-list-view-block-select-button__image"
```

Comment:
Do you reckon this would need some sort of `aria-label/description`?

Reply:
It's not an interactive element, and visually it's not really conveying any information that you can't already get from the block type, so it shouldn't need a description. The only thing I imagine _could_ be useful info here is the number of images in a Gallery, but I'm not sure if that sort of thing really has a place in the list view.

Reply:
Good question, and very happy for feedback on this, but my thinking for this PR was not to add to the descriptions for the list view row. The announcements for the list view are already pretty verbose, and I wasn't sure how much value it would have to describe these small images. We could add descriptive text to describe how many images are contained with a Gallery block, for example, but I think that would probably be better to explore in a separate PR since there might not be a 1:1 relation between what we want to have happen visually, and what we want to be described.

Reply:
@tellthemachines you beat me to it! 😄

Reply:
Sounds good! Thanks for the explainers.

Reply:
I've added an explicit `aria-hidden` so that it's clear(er) in code that it's intended to be hidden from screen readers.

Reply:
Good compromise, thanks!